### DISCUSSION TITLE
onBlur functionality?

### DISCUSSION THREAD

**Discussion by haveanicedavid** [outline/rich-markdown-editor#213] [General]
Hi there! This editor ie exactly what I was looking for, however there's one feature I need for my current functionality: onBlur (or something that functions the same - a callback that fires when the user navigates away from the editor).

Is this possible with the current features? If not, is there any plan to implement (or ways I could help implement?

**Comment by tommoor** [comment]
Howdy, nothing exposed currently in terms of onBlur/onFocus but they could be implemented for sure. Willing to accept a PR!
